doc/man1/openssl-srp.pod

```diff
@@ -20,7 +20,9 @@ B<openssl srp>
 [B<-userinfo> I<text>]
 [B<-passin> I<arg>]
 [B<-passout> I<arg>]
-[I<user> ...]
+[B<-rand> I<files>]
+[B<-writerand> I<file...>]
+[I<user...>]
```

Comment:
Why did you move the ellipsis inside the markup?

Reply:
habit.  fixed.